### DISCUSSION TITLE
www.distilledconsulting.com.au

### DISCUSSION THREAD

**Discussion by kbrookes** [joomlatools/joomlatools-pages#497] [Show and tell]
Hi all,

My first Pages site (built on 0.18, but will be upgraded soon) is Distilled Consulting:

https://www.distilledconsulting.com.au/

What I liked most in this process was not being bound to template overrides - being able to call any data from the database  really made experimenting with layout vastly easier than trying to shoe-horn it into a standard Joomla template structure.

I'll upgrade this to a later version of Pages soon and get the image optimisation working as well.

**Comment by alexandreelise** [comment]
Nice work
